File: TRICOUNT/TRICOUNT.java
import java.util.*;
import java.io.*;
class TRICOUNT{
    public static void main(String[] args){
        Scanner in = new Scanner(System.in);
        StringBuilder string = new StringBuilder();
        int num = in.nextInt();
        for(int x = 0; x < num; x++){
            long n = in.nextLong();
            string.append(((n*(n+2)*(2*n+1))/8) + "\n");
        }
        System.out.print(string);
    }
}

File: TRICOUNT/TRICOUNT.cpp
#include <iostream>
using namespace std;

int main(){
    int n;
    cin >> n;
    for(int x = 0; x < n; x++){
        long long num;
        cin >> num;
        cout << (num * (num + 2) * (2*num + 1))/8 << endl;
    }
    return 0;
}
